[Bug] On Android, only on Release build : java.lang.IllegalArgumentException: Couldn't find meta-data for provider with authority com.XXX.XXX.fileProvider

<!-- Bug report best practices: https://github.com/xamarin/Essentials/wiki -->

### Description
Exception is thrown when calling `await MediaPicker.CapturePhotoAsync();`
This exception is thrown just after permission checks and requests
The code works in debug mode, but not in **release mode**

Perhaps it's related to the necessary configuration detailed in MediaPlugin from James Montemagno:
[https://github.com/jamesmontemagno/MediaPlugin#android-required-setup](https://github.com/jamesmontemagno/MediaPlugin#android-required-setup)

But no reference is made to this kind of configuration in Xamarin.Essentials.MediaPicker's documentation
### Expected Behavior
Exception is not thrown in **Release mode**
### Actual Behavior
The following exception is thrown:
```
java.lang.RuntimeException: Unable to start activity ComponentInfo{com.XXX.XXXX/crc64a0e0a82d0db9a07d.IntermediateActivity}: java.lang.IllegalArgumentException: Couldn't find meta-data for provider with authority com.XXX.XXXX.fileProvider
at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3326)
at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3465)
at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:83)
at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2069)
at android.os.Handler.dispatchMessage(Handler.java:107)
at android.os.Looper.loop(Looper.java:214)
at android.app.ActivityThread.main(ActivityThread.java:7417)
at java.lang.reflect.Method.invoke(Native Method)
at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:980)
E/AndroidRuntime(28427): Caused by: java.lang.IllegalArgumentException: Couldn't find meta-data for provider with authority com.XXX.XXXX.fileProvider
at androidx.core.content.FileProvider.parsePathStrategy(FileProvider.java:606)
at androidx.core.content.FileProvider.getPathStrategy(FileProvider.java:579)
at androidx.core.content.FileProvider.getUriForFile(FileProvider.java:417)
at crc64a0e0a82d0db9a07d.IntermediateActivity.n_onCreate(Native Method)
at crc64a0e0a82d0db9a07d.IntermediateActivity.onCreate(IntermediateActivity.java:31)
at android.app.Activity.performCreate(Activity.java:7817)
at android.app.Activity.performCreate(Activity.java:7806)
at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1306)
at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3301)

```
### Basic Information

- Version with issue: v1.6.0-pre3
- Last known good version: none
- IDE: Visual Studio 2019 16.7.7
- Platform Target Frameworks: 
  - Android: 11.0 (API Level 30 - R) 
- Affected Devices: tested on two real devices (Gigaset GX290, Xiaomi MI MX 2S)

--- a/Xamarin.Essentials/Platform/Platform.android.cs
+++ b/Xamarin.Essentials/Platform/Platform.android.cs
@@ -414,7 +414,7 @@ namespace Xamarin.Essentials
             if (!string.IsNullOrEmpty(output))
             {
                 var javaFile = new Java.IO.File(output);
-                var providerAuthority = Platform.AppContext.PackageName + ".fileProvider";
+                var providerAuthority = FileProvider.Authority;
                 outputUri = FileProvider.GetUriForFile(Platform.AppContext, providerAuthority, javaFile);
                 actualIntent.PutExtra(MediaStore.ExtraOutput, outputUri);
             }

--- a/Xamarin.Essentials/Types/FileProvider.android.cs
+++ b/Xamarin.Essentials/Types/FileProvider.android.cs
@@ -28,6 +28,8 @@ namespace Xamarin.Essentials
         // We can choose external only, or internal only as alternative options
         public static FileProviderLocation TemporaryLocation { get; set; } = FileProviderLocation.PreferExternal;
 
+        internal static string Authority => Platform.AppContext.PackageName + ".fileProvider";
+
         internal static Java.IO.File GetTemporaryDirectory()
         {
             var root = GetTemporaryRootDirectory();

